layers/+tools/translate/packages.el

```diff
@@ -0,0 +1,80 @@
+;;; packages.el --- Translate Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Ray Wang <rayw.public@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+(defconst translate-packages
+  '(
+    (translate-mode :location (recipe :fetcher github :repo "rayw000/translate-mode"))
+    go-translate
+    popup
+    posframe
+    ))
+
+(defun translate/init-translate-mode ()
+  "Initialize required packages."
+  (use-package translate-mode
+    :demand t
+    :hook (translate-mode . set-transalte-mode-paragraph-functions)))
+
+(defun translate/init-go-translate ()
+  (use-package go-translate
+    :demand t
+    :config
+    (progn
+      (setq gts-translate-list '(("en" "zh")))
```

Comment:
The language pair should not be hard-coded into the layer since not everybody will need to use it. You have already mentioned in the README that `gts-translate-list` has to be specified as layer variables, so let users do so there.